Project.toml

```
@@ -1,7 +1,7 @@
 name = "SkyCoords"
 uuid = "fc659fc5-75a3-5475-a2ea-3da92c065361"
 authors = "Kyle Barbary, Mosé Giordano, and contributors"
-version = "1.4.0"
+version = "1.4.1"
```

There are new features: https://github.com/JuliaAstro/SkyCoords.jl/compare/JuliaAstro:0ab70e8...JuliaAstro:c8ae628
```suggestion
version = "1.5.0"
```